modules/tobira/src/main/java/org/opencastproject/tobira/impl/TobiraEndpoint.java

```diff
@@ -204,4 +214,31 @@ private static Response badRequest(String msg) {
     logger.warn("Bad request to tobira/harvest: {}", msg);
     return Response.status(BAD_REQUEST).entity(msg).build();
   }
+
+  @POST
+  @Path("/stats")
+  @RestQuery(
+      name = "stats",
+      description = "Accepts a json blob of statistical data about Tobira.",
+      restParameters = {
+          @RestParameter(
+              name = "stats",
+              isRequired = true,
+              description = "The statistics data generated by Tobira",
+              type = Type.STRING
+          )
+      },
```

Comment:
So this means using `application/x-www-form-urlencoded` correct? Would it be possible to accept an `application/json` body? I.e. the whole HTTP requests body is just JSON? Would be easier for Tobira and I don't see the need to put another encoding on top, i.e. I don't see why we would add additional form parameters. We can just add additional fields in the JSON instead.

Reply:
Try it now?  I *think* I've got it right...